internal/update/updates.go

```
@@ -386,13 +379,13 @@ func RetrieveUpdateStoredMetadata(update types.Update) (*types.UpdateStoredMetad
 // unique-key/constraint conflicts in relational stores, we append a deterministic
 // platform identifier digit (+1 for iOS, +2 for Android) to the end of the timestamp,
 // mathematically decoupling their identities under any hardware concurrency schedule.
-func GenerateUpdateTimestamp(platform string) int64 {
+func GenerateUpdateTimestamp(platform types.Platform) int64 {
 	milli := time.Now().UnixNano() / int64(time.Millisecond)
 	var platformModifier int64 = 0
```

golangci-lint (2.12.2)

[error] ineffectual assignment to platformModifier

(ineffassign)